File: backend/B2BApplication/DTO/LignePanierDto.cs
using B2BApplication.Models;

namespace B2BApplication.DTO
{
    public class LignePanierDto
    {
        public int id { get; set; }
        public int articleId { get; set; }
        public decimal tarifttc { get; set; }
        public string articleIntitule { get; set; }
        public string articleCode { get; set; }
        public string articleImage { get; set; }
        public decimal tarif { get; set; }
        public decimal tva { get; set; }
        public int ligneQuantite { get; set; }
        public decimal ligneTotalHt { get; set; }
        public decimal ligneTotalTtc { get; set; }
        public decimal ligneTotalTaxes { get; set; }
        public DateTime dateCreate { get; set; } = DateTime.Now;
        public string? userId { get; set; }
    }
}

File: frontend/src/app/models/LignePanierDto.ts
import { Article } from "./article";

export class LignePanierDto{
    id:number
articleId:number
tarif:number
tva:number
 articleIntitule:string 
articleCode:string
ligneQuantite:number
ligneTotalHt:number
ligneTotalTtc:number
ligneTotalTaxes:number
dateCreate:Date
tarifttc:number
articleImage=""
constructor(id:number,articleId:number,articleIntitule:string,articleCode:string,
    tarif:number,ligneQuantite:number,ligneTotalHt:number,ligneTotalTtc:number,ligneTotalTaxes:number,tva:number,dateCreate:Date,tarifttc:number){
this.id=id
this.articleId=articleId
this.tarif=tarif
this.ligneQuantite=ligneQuantite
this.ligneTotalHt=ligneTotalHt
this.ligneTotalTtc=ligneTotalTtc
this.ligneTotalTaxes=ligneTotalTaxes
this.dateCreate=dateCreate
this.tva=tva
this.articleIntitule=articleIntitule
this.articleCode=articleCode
this.tarifttc=tarifttc
    }
}